gptcli/config.py

```diff
@@ -22,6 +22,7 @@ class GptCliConfig:
     openai_api_key: Optional[str] = os.environ.get("OPENAI_API_KEY")
     anthropic_api_key: Optional[str] = os.environ.get("ANTHROPIC_API_KEY")
     google_api_key: Optional[str] = os.environ.get("GOOGLE_API_KEY")
+    default_editor: Optional[str] = os.environ.get("GPT_CLI_DEFAULT_EDITOR")
```

Comment:
```suggestion
    default_editor: Optional[str] = os.environ.get("VISUAL") or os.environ.get("EDITOR")
```

Reply:
I'm good with EDITOR env variable - I'd lean towards a default if not specified (i.e. current behavior)

Reply:
Same change implemented in a subsequent commit